online-inference/tensorizer-isvc/gptj_6b_hf/flask/transformer_flask.py

```diff
@@ -0,0 +1,68 @@
+from typing import List, Dict
+import logging
+import requests
+import numpy as np
+import base64
+import time
+import torch
+from flask import Flask
+from transformers import AutoTokenizer, GPTJForCausalLM
+
+# The signature name is defined at time of export, in signature_def_map supplied to builder
+# Tensorflows default is serving_default
+
+class Transformer(object):
+    def __init__(self):
+        
+        device = "cuda"
+        start = time.time()
+        self.model = GPTJForCausalLM.from_pretrained("/mnt/pvc", torch_dtype=torch.float16).to(device)
```

Comment:
Double-checking — should this specify a particular folder under `/mnt/pvc`?

Reply:
I guess a further question is: at what step would these be downloaded into the PVC, and where is that PVC defined? Maybe I missed something.

Reply:
Yes, the model was downloaded and stored using `save_pretrainined` in the /mnt/pvc path. Once we scope out docs and how to make this customer facing, we can make it a specific path